engine/flink/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/sink/flink/FlinkKafkaUniversalSink.scala

```diff
@@ -41,6 +41,7 @@ class FlinkKafkaUniversalSink(
       dataStream: DataStream[ValueWithContext[Value]],
       flinkNodeContext: FlinkCustomNodeContext
   ): DataStreamSink[_] =
+    // FIXME: Missing map TypeInformation
```

Comment:
You want to fix this in this PR or it is a TODO for someone else?

Reply:
I'll do it later -> next PR.